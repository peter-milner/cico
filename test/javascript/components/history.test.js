import React from 'react'
import { shallow } from 'enzyme'

import History from '../../../app/javascript/components/history'

const defaultProps = {
    events: []
}

const render = (props) => {
    const newProps = Object.assign({}, defaultProps, props)
    return shallow(
        <History {...newProps} />
    )
}

describe('<History />', () => {
    test('snapshot', () => {
        const rendered = render()
        expect(rendered).toMatchSnapshot()
    })
})
