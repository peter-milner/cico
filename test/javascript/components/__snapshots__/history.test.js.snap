// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<History /> snapshot 1`] = `ShallowWrapper {}`;
